setup.py

```diff
@@ -46,6 +46,10 @@ def get_version(root_path):
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Framework :: Django',
+        'Framework :: Django :: 1.8',
+        'Framework :: Django :: 1.9',
+        'Framework :: Django :: 1.10',
+        'Framework :: Django :: 1.11',
```

Comment:
https://github.com/jazzband/django-model-utils/issues/275  -- Seems there are still some issues pending for 1.11